### ISSUE TITLE
Show ip and port in the startup message

### ISSUE THREAD

**Issue by marco-buttu** [discos/simulators#274]
For instance:

```
$ discos-simulator start
Simulator 'backend' up and running at ('0.0.0.0', 12802).
Simulator 'gaia' up and running at ('0.0.0.0', 12600).
...
```

It is useful while performing manual tests and do not remember the ip and port.

**Comment by giuseppe-carboni** [comment]
Nice idea, it can also be helpful if the simulator type (if present) is showed:
```
$ discos-simulator start
Simulator 'backend, sardara' up and running at ('0.0.0.0', 12801).
Simulator 'backend, mistral' up and running at ('0.0.0.0', 12802).
Simulator 'if_distributor, IFD' up and running [...]
```
Something like this, up to you to choose how to display the simulator type

**Comment by marco-buttu** [comment]
Yes, that is perfect

**Comment by marco-buttu** [comment]
Fixed in commit [6b06cd6](https://github.com/discos/simulators/pull/275/commits/6b06cd64b14fa0718761cfb58a79ca0af80ca161).
